### PR TITLE
added footer code

### PR DESCRIPTION
**Pull request by shubhamku044**
image
Created footer

### CODE REVIEW

**Comment by EmpAsh938** [comment]
I would like to ask you make some changes.
- [ ] instead of lorem ipsum put (a modern blogging application where you can write and share blogs in public)
- [ ] Make year dynamic not manual (use Date())

File: frontend/src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                <GrYoutube className="text-3xl" />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div className="bg-black py-4">
+        <div>
+          <p className="text-center">
+            &copy; 2021 WriteIn. All rights reserved.
```
Comment (EmpAsh938) [inline frontend/src/components/Footer.tsx:39]: Here use Date()

File: frontend/src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { GrFacebook, GrTwitter, GrLinkedin, GrYoutube } from 'react-icons/gr'
+
+const Footer = () => {
+  return (
+    <div className="bg-white text-white border-t border-t-green-200">
+      <div className="max-w-2xl text-black mx-auto py-20">
+        <div className="flex items-center flex-col space-y-6 sm:space-y-8 px-8 sm:py-0">
+          <h6 className="font-semibold text-4xl sm:text-5xl">WriteIn</h6>
+          <p className="text-center sm:leading-8 sm:text-lg tracking-wide">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nobis illum non maiores mollitia nam nulla cum officiis tempore harum amet eligendi, at, eius voluptatem unde ipsum quibusdam, sapiente rem dolore?
```
Comment (EmpAsh938) [inline frontend/src/components/Footer.tsx:10]: A modern blogging application platform where you can write and share in public.

**Review by EmpAsh938 — APPROVED** [review]
I recorrected things I wanted and will be merged

**Comment by shubhamku044** [comment]
Is there anything else left for me?

**Pull request merged**